image/install-alpine.yaml

```diff
@@ -7,6 +7,22 @@
     - name: update alpine
       command: "apk upgrade --update-cache --available"
 
+    # use dhcpcd instead of udhcpd (for ipv6)
+    - name: install dhcpcd
+      command: "apk add dhcpcd"
+
+    - name:
+      ansible.builtin.lineinfile:
+        path: /etc/dhcpcd.conf
+        search_string: 'slaac private'
+        line: '#slaac private'
+
+    - name:
+      ansible.builtin.lineinfile:
+        path: /etc/dhcpcd.conf
+        search_string: 'slaac hwaddr'
+        line: 'slaac hwaddr'
```

Comment:
does this change something? i think you wanted to comment it out right?

Reply:
No this is correct like this. This replaces all lines which *contains* the search_string and replace it.
In this step `slaac hwaddr` will be the comment removed.

Reply:
this makes sense!